pdns/common_startup.hh

```diff
@@ -46,7 +46,7 @@ extern CommunicatorClass Communicator;
 extern std::shared_ptr<UDPNameserver> N;
 extern vector<std::shared_ptr<UDPNameserver> > g_udpReceivers;
 extern int avg_latency;
-extern TCPNameserver *TN;
+extern std::shared_ptr<TCPNameserver> TN;
```

Comment:
```suggestion
extern std::unique_ptr<TCPNameserver> TN;
```

Reply:
I made it a `shared_ptr` as the `UDPNameserver` is also a `shared_ptr`, but it might indeed be unique

Reply:
I haven't looked into why but `UDPNameserver` objects are apparently also stored in a vector so they can't be unique, but `TCPNameserver` ones should be fine.